File: LinkhedIn/src/query/job.ts
import { gql } from "@apollo/client";


export const FIND_JOB_QUERY = gql`
query FindJob{
  jobs{
     id
      title
      companyName
      location
      createdAt
      photoProfile
  }
}
`

export const CREATE_JOB_QUERY = gql`
mutation CreateJob($title:String!, $companyName:String!,$location:String!, $photoProfile:String!){
  createJob(input:{title:$title, companyName:$companyName, location:$location,photoProfile:$photoProfile})
}
`

File: backend/graph/model/job.go
package model

import "time"

type Job struct {
	ID           string    `json:"id"`
	Title        string    `json:"name"`
	CompanyName  string    `json:"company_name"`
	Location     string    `json:"location"`
	CreatedAt    time.Time `json:"created_at"`
	PhotoProfile string    `json:"photo_profile"`
}
